dafni_cli/model.py

```diff
@@ -1,15 +1,15 @@
 import click
```

Comment:
As model is starting to get fairly big, with the metadata code now too. Might be worth creating a model folder that all of this is nested in. I plan to take a similar approach with the datasets

Reply:
This will be addressed in next story on Models